evalml/data_checks/default_data_checks.py

```diff
@@ -16,6 +17,7 @@ class DefaultDataChecks(DataChecks):
     Includes:
 
         - `HighlyNullDataCheck`
```

Comment:
Maybe we should rename this to `HighlyNullColumnsDataCheck`? No need to handle in this pr. Probably best to issue a deprecation warning if we decide to do it.

Reply:
good call - I'll put up an issue for it 😄